shared/oclifAdapter.js

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
Oclif doesn't require a main hook defined in package.json. Instead, I moved that code to this adapter and deleted the index.js file.